ci/requirements.txt

```diff
@@ -4,3 +4,4 @@ pytest>=3.3.0
 tabulate>=0.8.2
 python-frontmatter>=0.4.2
 twisted==18.4.0
+attrs>=17.4.0
```

Comment:
Travis was showing an error like this when trying to import scrapy:
```TypeError: attrib() got an unexpected keyword argument 'converter'````

Googling the error turned up:
https://github.com/Julian/jsonschema/issues/449

Which said that older versions of the `attrs` library used an argument named `convert` instead of `converter`. That thread recommended having at least version 17.4.0 of `attrs` installed. Travis was showing that only 17.2.0 was installed (I think because one of the libraries we use listed that version as its dependency). Adding 17.4.0 as the minimum version in requirements.txt solved the issue in the next Travis builds.